README.md

```diff
@@ -654,6 +655,26 @@ Example configuration of environment variables
         value: foo
 ```
 
+#### Extra Settings
+
+With`extra_settings`, you can pass multiple custom settings via the `awx-operator`. The parameter `extra_settings`  will be appended to the `/etc/tower/settings.py` and be can an alternative to the `extra_volumes` parameter.
```

Comment:
@tchellomello nit pick here, I think this might be a typo:

```
and be can an alternative to the `extra_volumes` parameter.
```
-->
```
and can be an alternative to the `extra_volumes` parameter.
```

Reply:
Thanks @rooftopcellist++